lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -52,6 +53,8 @@ Creates object instance of the class
 
 =item * C<timeout> - Request timeout, in seconds. If not set, uses the environment variable C<RPC_QUEUE_RESPONSE_TIMEOUT>, or defaults to 300
 
+=item * C<category_map_config> - Category mapper configuration file path.
```

Comment:
we already have similar field in action config https://github.com/regentmarkets/binary-websocket-api/blob/master/lib/Binary/WebSocketAPI/Actions.pm#L243 why do we need to move it to separate yaml file?